host_modules/image_service.py

```diff
@@ -0,0 +1,70 @@
+"""
+Services related to SONiC images, such as:
+1) Download
+2) Install
+"""
+
+import logging
+import subprocess
+import os
+
+from host_modules import host_service
+
+MOD_NAME = "image_service"
+
+DEFAULT_IMAGE_SAVE_AS = "/host/downloaded-sonic"
```

Comment:
>host [](http://example.com/codeflow?start=26&length=4)

/tmp/ is more safe as a default file location.

Reply:
Done.